esp/esp/users/forms/user_profile.py

```diff
@@ -398,7 +399,7 @@ class TeacherProfileForm(UserContactForm, TeacherInfoForm):
     """ Form for teacher profiles """
     def __init__(self, *args, **kwargs):
         super(TeacherProfileForm, self).__init__(*args, **kwargs)
-        for field_name in Tag.getTag('teacher_profile_hide_fields', default='').split(','):
+        for field_name in [x.strip() for x in Tag.getTag('teacher_profile_hide_fields', default='').split(',')]:
```

Comment:
I would also recommend converting the field name to lowercase, to further normalize (double check that all the existing fields are lowercase).

Reply:
Yep, all of the existing fields for the TeacherInfoForm are lowercase, so I've gone ahead and added `.lower()`. Thanks, @hwatheod!